zio-http/src/test/scala/zhttp/middleware/MiddlewareSpec.scala

```diff
@@ -166,6 +166,52 @@ object MiddlewareSpec extends DefaultRunnableSpec with HttpAppTestExtensions {
               res <- app(request)
             } yield assert(res.getHeadersAsList)(hasSubset(expected))
           }
+      } +
+      suite("addCookie middleware") {
+        testM("should add set-cookie header") {
+          val app = (Http.ok @@ addCookie(Cookie("test", "testValue"))).getHeader("set-cookie")
+          assertM(app(Request()))(
+            equalTo(Some(Cookie("test", "testValue").encode)),
+          )
```

Comment:
```suggestion
          val cookie = Cookie("test", "testValue")
          val app = (Http.ok @@ addCookie(cookie)).getHeader("set-cookie")
          assertM(app(Request()))(equalTo(Some(cookie.encode)))
```

Similarly for the other cookie test.